config/puma.rb

```diff
@@ -47,3 +47,11 @@
   require 'prometheus_exporter/instrumentation'
   PrometheusExporter::Instrumentation::Puma.start(type: 'puma')
 end
+
+on_worker_shutdown do
+  puts '-------------------------Puma worker shutdown-------------------------'
```

Comment:
Is there a better alternative using puma's logger?

Reply:
We're not sure about the context here :thinking:

Reply:
Also this is just temporary

Reply:
Yeah, this is just temporary so it won't be staying but if we wanted it to be permanent I could try to think of something better?

Reply:
ack